.golangci.yml

```diff
@@ -17,4 +17,9 @@ linters:
     - staticcheck
     - unused
     - gosimple
-  
\ No newline at end of file
+    - gosec
+issues:
+  exclude-rules:
+    - linters:
+      - gosec
+      text: "G110|G601|G404|G204|G306"
```

Comment:
I am not in favor to add gosec because I have been testing it and in POV it has not raised nice things and raise a false positive as well.